com.bambulab.BambuStudio.yml

```diff
@@ -144,7 +144,7 @@ modules:
       - -DwxUSE_OPENGL=ON
       - -DwxUSE_WEBREQUEST=ON
       - -DwxUSE_WEBVIEW=ON
-      - -DwxUSE_REGEX=sys
+      - -DwxUSE_REGEX=builtin
```

Comment:
Why? This should be a separate commit with a description of why the change was made.